Lib/test/test_sys.py

```diff
@@ -1187,6 +1187,9 @@ def check(tracebacklimit, expected):
             b'    ~~^~~',
             b'ZeroDivisionError: division by zero'
         ]
+        self.assertTrue(hasattr(sys, "tracebacklimit"),
+                        'tracebacklimit attribute does not exist')
+        self.assertEqual(getattr(sys, 'tracebacklimit', 1_000), 1_000)
```

Comment:
```suggestion
        self.assertEqual(sys.tracebacklimit, 1_000)
```

We already know that the attribute exists.